docs/design-appconfig.html

```
@@ -186,47 +495,47 @@ <h1>Application Configuration Management Process</h1>
 <p>To ensure services can be reliably deployed and operated across multiple environments, we maintain a central registry of required configuration settings in <code>/app-config.json</code>. This file provides a clear inventory of what must be provisioned for the system to function correctly.</p>
-<h2 id="when-to-add-a-setting-to-app-config.json">When to Add a Setting to <code>/app-config.json</code> <a id="when-to-add-a-setting-to-app-config-json" href="#when-to-add-a-setting-to-app-config-json" class="markdown-heading-anchor text-decoration-none" aria-label="Link to section: When to Add a Setting to /app-config.json"><i class="bi bi-link-45deg" style="opacity: 0.2; font-size: 20px;"></i></a></h2>
+<h2 id="when-to-add-a-setting-to-app-config.json" class="content-heading">When to Add a Setting to <code>/app-config.json</code> <a id="when-to-add-a-setting-to-app-config-json" href="#when-to-add-a-setting-to-app-config-json" class="markdown-heading-anchor text-decoration-none" aria-label="Link to section: When to Add a Setting to /app-config.json"><i class="bi bi-link-45deg" style="opacity: 0.2; font-size: 20px;"></i></a></h2>
 <p>Developers MUST add a configuration key to <code>/app-config.json</code> if:</p>
 <ol>
 <li><strong>The setting is required</strong> for the service to start or perform its core functions.</li>
 <li><strong>The setting is environment-specific</strong> (e.g., URLs, connection strings, API keys) and does not have a &quot;one-size-fits-all&quot; default.</li>
 <li><strong>The setting is an override</strong> of a default value that is expected to change in most production or production-like environments.</li>
 <li><strong>The setting is security-sensitive</strong> (e.g., Auth Authority, Secret Keys). Note: only the <em>key</em> and <em>description</em> are added; never the <em>value</em>.</li>
 </ol>
-<h2 id="when-to-exclude-a-setting">When to Exclude a Setting <a id="when-to-exclude-a-setting" href="#when-to-exclude-a-setting" class="markdown-heading-anchor text-decoration-none" aria-label="Link to section: When to Exclude a Setting"><i class="bi bi-link-45deg" style="opacity: 0.2; font-size: 20px;"></i></a></h2>
+<h2 id="when-to-exclude-a-setting" class="content-heading">When to Exclude a Setting <a id="when-to-exclude-a-setting" href="#when-to-exclude-a-setting" class="markdown-heading-anchor text-decoration-none" aria-label="Link to section: When to Exclude a Setting"><i class="bi bi-link-45deg" style="opacity: 0.2; font-size: 20px;"></i></a></h2>
```

HTMLHint (1.9.2)

[error] The id value [ when-to-exclude-a-setting ] must be unique.

(id-unique)